bot/internal/bot/bot.go

```diff
@@ -129,21 +129,59 @@ func New(c *Config) (*Bot, error) {
 
 // classifyChanges determines whether the PR contains code changes
 // and/or docs changes.
-func classifyChanges(e *env.Environment, files []github.PullRequestFile) (docs bool, code bool, err error) {
+func classifyChanges(e *env.Environment, files []github.PullRequestFile) env.Changes {
+	ch := env.Changes{
+		Large:   !e.IsCloudDeployBranch() && xlargeRequiresAdminApproval(files),
+		Release: isReleasePR(e, files),
+	}
 	switch e.Repository {
 	case env.TeleportRepo:
 		for _, file := range files {
 			if strings.HasPrefix(file.Name, "docs/") ||
 				file.Name == "CHANGELOG.md" {
-				docs = true
+				ch.Docs = true
 			} else {
-				code = true
+				ch.Code = true
 			}
 		}
 	default:
-		code = true
+		ch.Code = true
+	}
+	return ch
+}
+
+// isReleasePR applies a number of heuristics to the PR changeset to determine
+// whether it's a release PR.
+func isReleasePR(e *env.Environment, files github.PullRequestFiles) bool {
+	// Check if the branch name starts with release/ prefix.
```

Comment:
This would be a new requirement, right?

Reply:
Yeah. Now that multiple people are going to be publishing, I thought it would make sense to standardize on the branch name rather than doing `roman/1.2.3` which I've been using before. I documented it in our Slab post about release process: https://gravitational.slab.com/posts/teleport-release-checklist-cf0s6vmt#hm0f0-quick-release-checklist.